src/main/java/tds/exam/results/configuration/ExamResultsTransmitterServiceProperties.java

```diff
@@ -68,6 +69,19 @@ public void setTisUrl(final String tisUrl) {
         this.tisUrl = tisUrl;
     }
 
+    /**
+     * Gets the callback url for the Test Integration System
+     *
+     * @return the TIS callback URL
+     */
+    public String getTisCallbackUrl() {
```

Comment:
This value is `http://tds-web-deployment.sbtds.org/student/tisReply` in our performance environment right now.  When TIS receives or scores a TRT it callsback to TDS to inform it and i think audit records are update.  I'm not sure if it is for scoring or just getting the result in the first place though.